Liroy/bootstrap.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+echo "Updating system"
+yum -y update
+yum -y install epel-release yum-utils
+yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo
+
+echo "Installing Docker"
+yum -y install docker docker-ce docker-ce-cli containerd.io
```

Comment:
docker and docker-ce are two editions for docker. You want to use/install only one of them.